### DISCUSSION TITLE
Tasmota as Window Shutter

### DISCUSSION THREAD

**Discussion by guydvir2** [arendst/Tasmota#11263] [Q&A] [answered]
Hi,
I have a 2 relay system that controls an AC motor that take an aluminum window shutter, that controls UP and DOWN ( ESP8266 NodeMCU and a double Relay module ).

I use a 3 state SWITCH ( what was originally installed when was operated using a SWITCH and AC power ).
I wish to use same SWITCH as inputs- but Tasmota's functionality is explained only using `Button` (momentary) and not a `switch`.

Is it possible to implement as `switch` ?

```
setoption80 1
Interlock 1,2
Interlock ON
shutterMode 1
```

**Comment by ascillato2** [comment]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.

Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)

**Comment by guydvir2** [comment, marked as answer]
Just to sum up:
configure `GPIO0` (Relay Up) and `GPIO9` (Relay Down) as `Switch` in "Configuration" --> "Configure Module"  , and it'll work just right, without any additional configuration.

GoodLuck
